elm.json

```diff
@@ -3,7 +3,7 @@
     "name": "PaackEng/elm-google-maps",
     "summary": "Type safe google maps implementation",
     "license": "BSD-3-Clause",
-    "version": "1.0.0",
```

Comment:
Don't bump the version here, we should do it in another pr/commit as well

Reply:
can you revert this change? I mean, the whole file change. this is a generated file, then if we add another dependency, the file will probably add the 4 spaces tab again :'(